packages/flutter/lib/src/services/text_input.dart

```diff
@@ -846,6 +847,127 @@ mixin TextSelectionDelegate {
 
   /// Whether select all is enabled, must not be null.
   bool get selectAllEnabled => true;
+
+  /// Cut current [selection] to Clipboard.
+  ///
+  /// If [cause] is [SelectionChangedCause.toolBar], the position of
+  /// [bringIntoView] to selection will be called and hide toolBar.
```

Comment:
Slight rewording:

If and only if [cause] is [SelectionChangedCause.toolbar], the toolbar will be hidden and the current selection will be scrolled into view.

Reply:
Oh and please use this rewording for the other methods below, too!